dropwizard-e2e/src/test/java/com/example/app1/App1Test.java

```diff
@@ -56,7 +57,7 @@ public void custom404OnViewRenderMissingMustacheTemplate() {
 
     @Test
     public void earlyEofTest() throws IOException, InterruptedException {
-        // Only eof test so we ensure it's false before test
+        // Only eof test so we ensure it's false setUp test
```

Comment:
Looks like there are multiple occurrences of the before/setUp swap 😄 

I'll assume you'll fix all of them, so I'll stop commenting when I see one.

Reply:
Rolled back